Domains/Smartmetering/README.md

```diff
@@ -64,6 +66,18 @@ Currently, the following Smart Metering features are available within the open s
 - **[GetGetFirmwareVersionResponse](./smartmeteringwebservices/GetGetFirmwareVersionResponse.md)** is a request which returns the response from the [GetFirmwareVersionRequest](./smartmeteringwebservices/GetFirmwareVersion.md).
 - **[ReplaceKeys](./smartmeteringwebservices/ReplaceKeys.md)** is a request to change the keys on a E-meter.
 - **[GetReplaceKeysResponse](./smartmeteringwebservices/GetReplaceKeysResponse.md)** is a request which returns the response from the [ReplaceKeys](./smartmeteringwebservices/ReplaceKeys.md) request.
+- **[GetMbusEncryptionKeyStatusByChannel](./smartmeteringwebservices/GetMbusEncryptionKeyStatusByChannel.md)** is a request to get the M-Bus encryption key status from an M-Bus device by using the E-meter device identification and channel from the G-meter.
```

Comment:
I think it would look more logical if the "by-channel" operations were located right below the related "regular" version (by identification) instead of listing them separated by other operations.

(I noticed operations are called requests on this page, which looks somewhat confusing to me. This page describes *operations* that have an input/request and an output/response message, but the phrasing used is "operation-name is a *request* to ...".)